<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BF_ArchitectForm</class>
 <widget class="QWidget" name="BF_ArchitectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>841</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Проектирование</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../architect_resources.qrc">
    <normaloff>:/architectplugin/img/plugin.png</normaloff>:/architectplugin/img/plugin.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
        <normaloff>:/images/raznoe31/raznoe31/48x48/127.png</normaloff>:/images/raznoe31/raznoe31/48x48/127.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Добавление связи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QPushButton" name="setLinkButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Установить связь</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
           <normaloff>:/images/SilkCompanion1/icons/internet_link_02_9461.ico</normaloff>:/images/SilkCompanion1/icons/internet_link_02_9461.ico</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Описание</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Описание связи:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nameLineEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Название связи:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTextEdit" name="linkDescriptionEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Соединение</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="4" column="0">
            <widget class="QComboBox" name="linkTypeBox">
             <item>
              <property name="text">
               <string>Ассоциация</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Агрегация</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Композиция</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Количество</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="linkACountBox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Связь А</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="linkAEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Связь Б</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Количество</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="linkBCountBox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="linkBEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="resetLinkAButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Сбросить соединение</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
               <normaloff>:/images/raznoe31/raznoe31/48x48/118.png</normaloff>:/images/raznoe31/raznoe31/48x48/118.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="resetLinkBButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Сбросить соединение</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
               <normaloff>:/images/raznoe31/raznoe31/48x48/118.png</normaloff>:/images/raznoe31/raznoe31/48x48/118.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="setLinkAButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Выбрать A точку</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
               <normaloff>:/images/raznoe6/raznoe6/Black pin.png</normaloff>:/images/raznoe6/raznoe6/Black pin.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="setLinkBButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>Выбрать B точку</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
               <normaloff>:/images/raznoe6/raznoe6/Black pin.png</normaloff>:/images/raznoe6/raznoe6/Black pin.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
        <normaloff>:/images/raznoe31/raznoe31/48x48/126.png</normaloff>:/images/raznoe31/raznoe31/48x48/126.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Удаление связи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" colspan="4">
        <widget class="QListWidget" name="linkListWidget">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="updateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Обновить</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
           <normaloff>:/images/raznoe31/raznoe31/48x48/48.png</normaloff>:/images/raznoe31/raznoe31/48x48/48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="deleteLinkButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Удалить</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
           <normaloff>:/images/raznoe31/raznoe31/48x48/118.png</normaloff>:/images/raznoe31/raznoe31/48x48/118.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Идетификатор</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="addArchitectElementButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/112.png</normaloff>:/images/raznoe31/raznoe31/48x48/112.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="editArchitectElementButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/31.png</normaloff>:/images/raznoe31/raznoe31/48x48/31.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTreeWidget" name="architectTreeWidget">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="autoExpandDelay">
      <number>-1</number>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>600</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>600</number>
     </attribute>
     <column>
      <property name="text">
       <string>Структура программы</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="editorButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/13.png</normaloff>:/images/raznoe31/raznoe31/48x48/13.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="deleteArchitectElementButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/118.png</normaloff>:/images/raznoe31/raznoe31/48x48/118.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../architect_resources.qrc"/>
  <include location="../../../../klever/core/frameWork/resources/framework_resources.qrc"/>
 </resources>
 <connections/>
</ui>
